README.md

```diff
@@ -36,13 +36,15 @@ https://readwriteexercise.com/posts/trust-self-signed-certificates-macos/
 
 To start:
 
-1. Do `npm run dev` in one terminal. You'll be able to view the development build in your browser at http://localhost:8000/
+1. Enter `nvm use` in the terminal to ensure that you are using the node version specified in .nvmrc.
```

Comment:
Made some minor updates to the readme, after onboarding Jen I noticed these 😂

Reply:
Nice, thanks for the updates.